docs/mcd.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Connection Dialog v2
---- 
-**IMPORTANT:** This feature is only available in Tableau 2020.3 or later.
+title: Build the Connection Dialog with Connection Dialog v2
+---
+**IMPORTANT:** This feature is only available in Tableau 2020.3 or later. For compatibility with older versions, use the legacy connection dialogs (documentation found [here]({ site.baseurl }}/docs/ui)).
```

Comment:
Call it Connection Dialog V1?

Reply:
+1, its still supported so not legacy.

Reply:
Please change "This feature is only available in" to "This feature is available only in". (Only modifies set of versions, not "available".) Thanks.

Reply:
Updated, thank you! Sorry for the long delay, this pull request completely slipped my mind.